cerberus/models/service.py

```diff
@@ -33,3 +33,9 @@ def __str__(self) -> str:
 
     def get_absolute_url(self) -> str:
         return reverse("service_detail", kwargs={"pk": self.pk})
+
+    def length_seconds(self) -> int:
+        return int(self.length.total_seconds())
+
+    def length_minutes(self) -> int:
+        return self.length_seconds() // 60
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_refinement):** Redundant method for calculating length in minutes.

Given that `length` is already a `DurationField`, consider using Django's built-in methods to get the duration in minutes directly, rather than converting seconds to minutes manually.